kaldb/src/main/java/com/slack/kaldb/chunk/ChunkInfo.java

```diff
@@ -66,77 +66,84 @@ public void setChunkSize(long chunkSize) {
   // Size of the chunk
   private long chunkSize;
 
-  public ChunkInfo(String chunkId, long chunkCreationTimeEpochSecs) {
+  public ChunkInfo(String chunkId, long chunkCreationTimeEpochMs) {
```

Comment:
There is a call site for this constructor that should also be changed: https://github.com/slackhq/kaldb/blob/master/kaldb/src/main/java/com/slack/kaldb/chunk/ReadWriteChunkImpl.java#L63